### ISSUE TITLE
Route [login] not defined when password is changed

### ISSUE THREAD

**Issue by EriksonEnterprises** [laravel-filament/filament#7725]
### Package

filament/filament

### Package Version

v3.0

### Laravel Version

v10.0

### PHP Version

v8.2.7

### Problem description

When you change a user's password either through the database directly, or through CRUD (inside of Filament), that user will then be taken to an error of "Route [login] not defined"

When they refresh the page it takes them to the login page like normal

The URL is correct (project.com/admin) when this occurs, but for some reason they have to refresh that URL to get the login page

This seems to be an error where the password doesn't match anymore so the system tries to "kick" the user, but fails to direct to the proper route. If you manually set the login route to "/admin" in web.php, this error never happens. But then you can't set this route for other panels (/user, /app, /owner, etc)

### Expected behavior

When changing a user's password, they should either stay logged in, or get kicked to the actual login page for the specific panel

### Steps to reproduce

Change the user's password in database or through CRUD and then refresh the user's panel (or click on a page/link, like Dashboard)

### Reproduction repository

https://github.com/filamentphp/filamentphp.com

**Comment by EriksonEnterprises** [comment]
https://github.com/filamentphp/filamentphp.com

**Comment by dpenacton** [comment]
🛠 **Trouble with Export/Import Functionality in Laravel Filament Admin Panel: 'Login Route Not Found' Error**

I've been tackling the export/import feature, aiming for smooth functionality. However, upon attempting to export or import a CSV, it seems to generate within the notifications. Yet, when I endeavor to download the file, a persistent 'login route not found' error arises.

![Screenshot](https://img.enacton.com/ShareX/2024/04/w4FMvKPUQJ.png)

I've configured separate guards for authentication, catering to both admin and user panels. Additionally, I've ensured that the export/import policy is correctly set to return true within the view function.

Furthermore, I've diligently included `Export::polymorphicUserRelationship();` in the app service provider and executed all requisite database changes.

Despite these efforts, I find myself still grappling with the 'login route not found' issue when attempting to access export/import. Any suggestions on resolving this within the Laravel Filament admin panel?
